src/sentry/static/sentry/app/views/performance/transactionSummary/sidebarCharts.tsx

```diff
@@ -141,35 +143,57 @@ function SidebarCharts({api, eventView, organization, router}: Props) {
   };
   const project = eventView.project;
   const environment = eventView.environment;
+  const threshold = organization.apdexThreshold;
 
   return (
     <RelativeBox>
-      <ChartTitle top="0px" key="apdex">
-        {t('Apdex')}
-        <QuestionTooltip
-          position="top"
-          title={getTermHelp(organization, PERFORMANCE_TERM.APDEX)}
-          size="sm"
-        />
-      </ChartTitle>
+      <ChartLabel top="0px">
+        <ChartTitle>
+          {t('Apdex')}
+          <QuestionTooltip
+            position="top"
+            title={getTermHelp(organization, PERFORMANCE_TERM.APDEX)}
+            size="sm"
+          />
+        </ChartTitle>
+        {totals ? (
+          <ChartValue>{totals[`apdex_${threshold}`].toFixed(4)}</ChartValue>
+        ) : (
+          <Placeholder height="24px" />
+        )}
+      </ChartLabel>
 
-      <ChartTitle top="150px" key="failure-rate">
-        {t('Failure Rate')}
-        <QuestionTooltip
-          position="top"
-          title={getTermHelp(organization, PERFORMANCE_TERM.FAILURE_RATE)}
-          size="sm"
-        />
-      </ChartTitle>
+      <ChartLabel top="160px">
+        <ChartTitle>
+          {t('Failure Rate')}
+          <QuestionTooltip
+            position="top"
+            title={getTermHelp(organization, PERFORMANCE_TERM.FAILURE_RATE)}
+            size="sm"
+          />
+        </ChartTitle>
+        {totals ? (
+          <ChartValue>{formatPercentage(totals.failure_rate)}</ChartValue>
+        ) : (
+          <Placeholder height="24px" />
+        )}
+      </ChartLabel>
 
-      <ChartTitle top="300px" key="throughput">
-        {t('TPM')}
-        <QuestionTooltip
-          position="top"
-          title={getTermHelp(organization, PERFORMANCE_TERM.TPM)}
-          size="sm"
-        />
-      </ChartTitle>
+      <ChartLabel top="320px">
+        <ChartTitle>
+          {t('TPM')}
+          <QuestionTooltip
+            position="top"
+            title={getTermHelp(organization, PERFORMANCE_TERM.TPM)}
+            size="sm"
+          />
+        </ChartTitle>
+        {totals ? (
+          <ChartValue>{tct('[tpm] tpm', {tpm: totals.tpm.toFixed(0)})}</ChartValue>
```

Comment:
We typically format TPM with up to 4 decimal places. You can see an example of this on the performance landing page.

Reply:
I'd argue that it's a bit dubious whether we should, but I'll change it to 4 for now for consistency.